docs/supported-tools.rst

```diff
@@ -0,0 +1,246 @@
+In the past we used to set these dependencies to a specific version and update them after some time,
+but by doing so would break some builds and new projects wouldn't be able to use a new version by default.
+For this reason we are now installing their latest version (or latest supported version) by default.
+
+.. note::
+
+   In order to keep your builds reproducible,
+   it's highly recommended declaring its dependencies and versions explicitly.
+
+   .. TODO: link to this guide once it's written https://github.com/readthedocs/readthedocs.org/issues/7852.
+
+External dependencies (Python)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Python environment:
+
+Sphinx:
+  Projects created before Oct 20, 2020 use ``1.8.x``.
+  New projects use the latest version.
+
+Mkdocs:
+  Projects created before April 3, 2019 (April 23, 2019 for :doc:`/commercial/index`) use ``0.17.3``.
+  New projects use the latest version.
+
+sphinx-rtd-theme:
+  Projects created before October 20, 2020 (January 21, 2021 for :doc:`/commercial/index`) use ``0.4.3``.
+  New projects use the latest version.
+
+pip:
+  Latest version by default.
+
+setuptools:
+  Latest version by default.
+
+mock:
+  ``1.0.1`` (could be removed in the future).
+
+pillow:
+  ``5.4.1`` (could be removed in the future).
+
+alabaster:
+  ``0.7.x`` (could be removed in the future).
+
+commonmark:
+  ``0.8.1`` (could be removed in the future).
+
+recommonmark:
+  ``0.5.0`` (could be removed in the future).
+
+External dependencies (Conda)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Conda environment:
+
+Conda:
+   Miniconda2 ``4.6.14``
+   (could be updated to use the latest version by default).
+
+Mkdocs:
+  Latest version by default installed via ``conda``.
+
+Sphinx:
+  Latest version by default installed via ``conda``.
+
+sphinx-rtd-theme:
+  Latest version by default installed via ``conda``.
+
+mock:
+  Latest version by default installed via ``pip`` (could be removed in the future).
+
+pillow:
+  Latest version by default installed via ``pip`` (could be removed in the future).
+
+recommonmark:
+  Latest version by default installed via ``conda`` (could be removed in the future).
+
+Internal dependencies
+~~~~~~~~~~~~~~~~~~~~~
+
+Internal dependencies are needed to integrate your docs with Read the Docs.
+We guarantee that these dependencies will work with all current supported versions of our tools,
+you don't need to specify them in your requirements.
+
+- readthedocs-sphinx-ext
+
+Table of supported versions
+---------------------------
+
+Sphinx
+~~~~~~
+
+Sphinx releases its versions incrementally.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Version
+     - Released / Latest Update
+     - Supported Until
+
+   * - ``3.x``
+     - Apr 5, 2020 / ???
+     - 5.0 is released or later/early
```

Comment:
What does "or later/early" mean? It seems like we don't really know :smile:

Reply:
This is a date that isn't decided yet, but are aiming to 5.0, but something can happen, and we should be ready to stop supporting this version or keep supporting it.

Reply:
This should be clarified in some manner in the policy.